src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+    public Order add(Order order);
+
+    public List<Order> getByUser(User user);
```

Comment:
remove public - interface methods are public by default

this is not the first time when you have such a comment